docs/architecture/README-copy.md

```diff
@@ -0,0 +1,303 @@
+# Overview
+
+This document describes the high-level architecture of nearcore. The focus here
+is on the implementation of the blockchain protocol, not the protocol itself.
+For reference documentation of the protocol, please refer to
+[nomicon](https://nomicon.io/)
+
+Some parts of our architecture are also covered in this [video series on YouTube](https://www.youtube.com/playlist?list=PL9tzQn_TEuFV4qlts0tVgndnytFs4QSYo).
+
+## Bird's Eye View
+
+If we put the entirety of nearcore onto one picture, we get something like this:
```

Comment:
If my design document existed as a PR, then I can leave inline comments on it at arbitrary points of the doc like this.

Reply:
Note that this mechanism is not restricted to docs that are newly created.  I can create a PR to modify an existing doc and then add comments to them as well.